docs/src/gcode/g-code.adoc

```diff
@@ -1392,6 +1392,9 @@ G43 <H->
 
 'G43 Hn' uses the offset for tool n.
 
+The active tool length compensation values are stored in the numbered
+parameters '5401-5409'.
```

Comment:
I think there should be no newlines in this hunk, to support translations.

Reply:
@smoe what is your experience here?

Reply:
> I think there should be no newlines in this hunk, to support translations.

My view of the matter is that. That translations should not hinder improvement.